spec.emu

```diff
@@ -842,15 +842,15 @@ contributors: Ujjwal Sharma, Younies Mahmoud
 
           <p>The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints for all locale values _locale_:</p>
 
-      <ul>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]].[[nu]] must be a List as specified in <emu-xref href="#sec-intl.numberformat-internal-slots"></emu-xref>.</li>
```

Comment:
This list item shouldn't be removed. Actually it'd might even be better to explicitly list the allowed values to match ECMA-402, cf. 
- https://tc39.es/ecma402/#sec-intl.datetimeformat-internal-slots
- https://tc39.es/ecma402/#sec-intl.numberformat-internal-slots
- https://tc39.es/ecma402/#sec-Intl.RelativeTimeFormat-internal-slots

In all cases the disallowed values ("native", "traditio", and "finance") are mentioned explicitly.

Reply:
I assumed that "as specified in ..." would make for more future-proof spec text but I understand that being explicit here is not a bad idea. I'll add it back.